specification/sql/resource-manager/readme.md

```diff
@@ -168,7 +168,7 @@ input-file:
 - Microsoft.Sql/stable/2014-04-01/serviceObjectives.json
 - Microsoft.Sql/stable/2014-04-01-legacy/sql.core_legacy.json
 - Microsoft.Sql/stable/2014-04-01-legacy/usages_legacy.json
-- ./Microsoft.Sql/preview/2020-11-01-preview/BlobAuditing.json
+- ./Microsoft.Sql/preview/2021-11-01-preview/BlobAuditing.json
```

Comment:
can we move it to the bottom? or sort it by API version?

Reply:
can we also update the PR title? 
I guess the title might be: Add 2021-11-01-preview BlobAuditing.json in SQL V5 SDK tag